core-tests/jvm/src/test/scala-2.12/zio/StacktracesSpec.scala

```diff
@@ -54,7 +54,7 @@ object StackTracesSpec extends DefaultRunnableSpec {
     },
     testM("foreachPar fail") {
       val io = for {
-        trace <- TestClock.setTime(1.second) *> foreachParFail
+        trace <- TestClock.setTime(fromSeconds(1)) *> foreachParFail
```

Comment:
Let's have extension methods in `zio.duration._` so we can still use the same syntax as before. Will help ensure people don't have to change their code.

Reply:
Good point, we can still use extension methods.